doc/features.md

```diff
@@ -0,0 +1,74 @@
+Network Module Features
+-----------------------
+
+Network Services
+======================
+
+- wicked
+- Network Manager
+- no network service
+
+IPv6
+==============
+
+- enable/disable it
+
+DHCP Client options
+====================
+
+- set hostname according to dhcp response
```

Comment:
i don't fully understand this note. If we talk about "Hostname to Send" then this is about hostname which is being send back to dhcp server.